backend/linkbox/linkbox.go

```diff
@@ -238,6 +239,14 @@ func (f *Fs) listAll(ctx context.Context, dirID string, name string, fn listAllF
 		// If name isn't good then do an unbounded search
 		name = ""
 	}
+
+	nameSearch := name
```

Comment:
Instead of doing this you can just limit the maximum length that is OK in the `searchOK` regexp above

Something like

```go
var searchOK = regexp.MustCompile(`^[a-zA-Z0-9_ .]{1,50}$`)
```

Reply:
Thank you for your comment. Yes, you're right, it's more concise.